state-network.md

```diff
@@ -255,53 +269,297 @@ sedes      := Container(enrs: List[byte_list, max_length=32], payload: byte_list
 > A response with an empty `payload` and empty `enrs` indicates that the node is not aware of any closer nodes, *nor* does the node have the requested content.
 
 
-#### Advertise (0x07)
+#### Offer (0x07)
 
-Advertise a set of content keys that this node has proofs available for.
+Offer a set of content keys that this node has proofs available for.
 
 ```
 message_id := 7
 type       := request
-sedes      := List[byte_list, max_length=32]
+sedes      := List[byte_list, max_length=64]
 ```
 
 The payload of this message is a list of encoded `content_key` entries.
 
 
-#### RequestProofs (0x08)
+#### Accept (0x08)
 
-Response message to Advertise (0x07).
+Response message to Offer (0x07).
 
-Request the proofs for a piece of content.
+Signals interest in receiving the offered data fro the corresponding Offer message.
 
-> Despite the name of this message having the name "Request" in it, this is a response message.
 
 ```
 message_id := 8
 type       := response
-sedes      := Container(connection_id: bytes4, content_keys: List[byte_list, max_length=32]]
+sedes      := Container(connection_id: bytes4, content_keys: BitList[max_length=64]]
 ```
 
 * `connection_id`: ConnectionID to be used for a uTP stream
     * ConnectionID values should be randomly generated.
-* `content_keys`: The set of encoded content keys that should be sent.
-    * Only content keys from the corresponding Advertise message are valid.
+* `content_keys`: Signals which content keys are desired
+    * A bit-list corresponding to the offered keys with the bits in the positions of the desired keys set to `1`.
 
 Upon *sending* this message, the requesting node should *listen* for an incoming uTP stream with the generated `connection_id`.
 
 Upon *receiving* this message, the serving node should initiate a uTP stream.
```

Comment:
I think there is some background about dropping `Store` - https://github.com/ethereum/trin/pull/84#issuecomment-905772594

Reply:
Yes, looking at the different payloads that need to be gossiped, the dominant majority of the data doesn't fit into a single packet so the STORE message would very rarely be a viable way to transmit data.  Removing it seemed to simplify the protocol in a nice way, especially since the message triplicate doesn't fit well within the request/response paradigm of discv5